src/Avalonia.Controls/ComboBox.cs

```diff
@@ -95,82 +95,89 @@ static ComboBox()
         {
             ItemsPanelProperty.OverrideDefaultValue<ComboBox>(DefaultPanel);
             FocusableProperty.OverrideDefaultValue<ComboBox>(true);
-            SelectedItemProperty.Changed.AddClassHandler<ComboBox>((x, e) => x.SelectedItemChanged(e));
-            KeyDownEvent.AddClassHandler<ComboBox>((x, e) => x.OnKeyDown(e), Interactivity.RoutingStrategies.Tunnel);
```

Comment:
I removed this as it seems to duplicate the OnKeyDown override itself. I think this was a relic from older versions of Avalonia that did not have OnKeyDown() method internally and it was never noticed until now.